lib/utils/hash/hash.ts

```diff
@@ -0,0 +1,9 @@
+import * as hash from 'object-hash';
```

Comment:
I'd suggest to rename `hash.ts` to `index.ts` or just implement directly as `lib/utils/hash.ts`